### ISSUE TITLE
NVDA crashes office applications when trying to open a document using the Google Drive Office add-in

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#7850]
### Steps to reproduce:
1. Install [the Google Drive plugin for Office](https://tools.google.com/dlpage/driveforoffice?hl=en)
2. Start Word or Excel 2013/2016
3. Make sure you are logged in inside the Google Drive add-in. In word 2013, I had to execute step 4 in order to get the login prompt. In Word 2016, the add-in asked me to login upon first launch.
4. On the ribbon, choose the Google Drive tab and select open.

### Expected behavior:
The Google Drive open window is successfully opened en you can open a file.

### Actual behavior:
NVDA crashes Word or Excel entirely. Unfortunately, appModule.dumpOnCrash() does not yield any crash dump when the Office application crashes. Also, no meaningful information is logged.

### System configuration:
NVDA version: 2017.4
NVDA Installed or portable: Installed
Windows version: Both windows 8.1 and 10 fall creators update
Name and version of other software in use when reproducing the issue: Office 2013/2016

### Other questions:
Does the issue still occur after restarting your PC? Yes
Have you tried any other versions of NVDA? Yes, latest Next also results in this issue.

**Comment by Adriani90** [comment]
@leonardder is this issue still reproducible?

**Comment by Adriani90** [comment]
@leonardder I tried in NVDA 2019.3 Beta 2 but I cannot reproduce it. are you still having this issue?

**Comment by leonardder** [comment]
I no longer need this particular setup, so if you say it works for you, I think it's safe to close this. We can always reopen.
